addons/calendar/models/calendar.py

```diff
@@ -529,6 +529,14 @@ class Meeting(models.Model):
     _inherit = ["mail.thread"]
 
     @api.model
+    def default_get(self, fields):
```

Comment:
You should not set a value for res_model and res_id if it has already been set in the super. Indeed you have to keep support of default_res_model, active_model should not override it but improve it.